backend/src/main/java/com/empire/ArtificialIntelligence.java

```diff
@@ -1021,13 +1021,14 @@ boolean feasible() {
 		FeedPeople() {
 			int turnsUntilHarvest = 4 - world.date % 4;
 			double totalStocks = 0;
-			double totalDemand = 0;
+			double totalDemand = 0.001;  // prevent a divide by zero with no citizens
```

Comment:
Good catch, but maybe better to just modify 1031 to be ` = totalDemand == 0 ? Number.POSITIVE_INFINITY : totalStocks / (totalDemand * turnsUntilHarvest)`

Reply:
Done.